package.json

```diff
@@ -12,6 +12,7 @@
     "compile": "npm run compile:stage-01 && npm run compile:stage-02 && npm run compile:stage-03",
     "compile:clean": "npm run compile:stage-01 -- --clean && npm run compile:stage-02 && npm run compile:stage-03 -- --clean",
     "watch": "npm run compile && tsc --build tsconfig.json --watch",
+    "version": "npm i",
```

Comment:
I understand why the re-install on `version` (which is heavy _-ish_) bump solves the out of sync `package-lock-only.json` in a way that doesn't require manipulating it ourselves but does this still work if we merely do `npm install --package-lock-only`?  

Seems a bit lighter weight?

If not, all good.

Reply:
💯